src/openepd/model/specs/singular/concrete.py

```diff
@@ -183,3 +183,16 @@ class ConcreteV1(BaseOpenEpdHierarchicalSpec):
     _csa_exposure_classes_exclusive_groups_validator = pyd.validator("csa_exposure_classes", allow_reuse=True)(
         exclusive_groups_validator_factory(CsaExposureClass)
     )
+
+    @pyd.root_validator(skip_on_failure=True)
+    def _validate_strength_other(cls, values: dict) -> dict:
+        strength_other = values.get("strength_other")
```

Comment:
Missing "strength_other" key and `None` value means different things when `PATCH` request. Are you taking this into account?

Reply:
uh, okay. I perceived the `strength_other` and `strength_other_d` as a single field by meaning, and would expect this to come always in pairs. 

However, if only one should be supplied, I will move this to the other level and validate at the object before the update (ensure that invariant holds that if strength then days).